tests/unit/cluster/failover2.tcl

```diff
@@ -64,7 +64,7 @@ start_cluster 3 4 {tags {external:skip cluster} overrides {cluster-ping-interval
     }
 } ;# start_cluster
 
-start_cluster 7 3 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 5000}} {
+start_cluster 7 3 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 9000}} {
```

Comment:
Let's adjust it back to 15 seconds, so it returns to the previous delay = 500ms, which was the default time we expected to resolve conflicts when writing the test case.

```suggestion
start_cluster 7 3 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 15000}} {
```

Reply:
I actually prefer to specify it explicitly, but either way is fine.

Reply:
please also fix the DCO